blog/_src/posts/2017-06-05-trip-report-pliss-2017.md

```diff
@@ -0,0 +1,130 @@
+    Title: Report: PLISS 2017
+    Date: 2017-06-05T15:47:59
+    Tags: pliss, event, by Ming-Ho Yee
+
+Two weeks ago, I attended the first [Programming Language Implementation Summer
+School][PLISS], held in beautiful Bertinoro, Italy.
+
+The goal of PLISS was "to prepare early graduate students and advanced
+undergraduates for research in the field," and I think it successfully
+accomplished that. There were many talks in a variety of areas, such as
+just-in-time compilers, garbage collection, static analysis, and distributed
+systems. But PLISS was more than just a series of talks: PLISS provided an
+environment for interacting with other students, as well as senior researchers.
```

Comment:
I'm not one of these people with, you know, very strict opinions about where commas should be placed, but I think this last one should be removed.